### PR TITLE
Features/debian11

### PR DESCRIPTION
**Pull request by ultraheroe**
### Description of the Change

Update role to install Debian 11
General role update

### Benefits

Debian 11 support

### Possible Drawbacks

Not known

### Applicable Issues

#7

### CODE REVIEW

**Review by frantsao — APPROVED** [review]

**Review by emepege — APPROVED** [review]

**Review by xtianae7 — APPROVED** [review]

**Review by vicsufer — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
-dist: xenial
+
+dist: focal
 language: python
-python: "3.7"
+python: "3.8"
```
Comment (ablancolopez) [inline .travis.yml]: 3.9 ;)

File: Pipfile [hunk 2]
```diff
@@ -6,10 +6,10 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-ansible = "==2.9.9"
-molecule = "==3.0.4"
+ansible = "==2.9.21"
```
Comment (ablancolopez) [inline Pipfile:9]: I'd say we can start testing in newer versions, see the [recently updated nginx role for example](https://github.com/idealista/nginx_role/blob/master/Pipfile)

File: molecule/cron-create-config/molecule.yml [hunk 3]
```diff
@@ -1,22 +1,31 @@
...
   ansible-lint
+
 platforms:
-  - name: logrotate
+  - name: logrotate-cron-create-config
     groups:
       - logrotate
     image: ${MOLECULE_DISTRO:-debian:stretch-slim}
-    privileged: true
+    privileged: false
```
Comment (ablancolopez) [inline molecule/cron-create-config/molecule.yml:18]: <3

**Review by ablancolopez — APPROVED** [review]

**Review by jmonterrubio — APPROVED** [review]

File: Pipfile (hunk 2 above)
Comment (ultraheroe) [reply Pipfile:9]: Annotated for future releases / other roles

File: .travis.yml (hunk 1 above)
Comment (ultraheroe) [reply .travis.yml]: Done!

**Pull request merged**